en/docs/references/enterprise-login.md

```diff
@@ -4,11 +4,13 @@ Choreo's Enterprise Login feature allows your users residing in an external IdP
 
 This guide takes you through the steps you need to follow to configure an enterprise login for your organization in Choreo. 
 
-Let's begin!
+
+## Prerequisites 
+Make sure you have a valid email domain. 
 
 ## Step 1: Create an organization in Choreo
 
-- Sign in to the Choreo Console at <a>https://console.choreo.dev/</a>. using a Google/ GitHub/ Microsoft account.
+- Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
   
       If you are a:
```

Comment:
The bullet points after this are not rendering properly.

Reply:
I think it's fine now.